src/App.vue

```diff
@@ -1,28 +1,106 @@
 <template>
-  <div id="app">
-    <img alt="Vue logo" src="./assets/logo.png">
-    <HelloWorld msg="Welcome to Your Vue.js App"/>
+  <div id="app" class="container">
+    <div class="input-group input-group-lg bottom">
+      <div class="input-group-prepend">
+        <span class="input-group-text">Search</span>
+      </div>
+      <input type="text" 
+        class="form-control col-md-6" 
+        @keyup.prevent="search"
+        v-model="query" />
+    </div>
+    
+    <div  class="card-row">
+    <div v-if="data" v-for="(value, index) in data" 
+      :key="index"
+      :ref="`card_${index}`"
+         class="card">
+      
+      <img class="card-image" :src="value._source.url">
+      
+      <div class="card-footer">
+        <h3 class="card-title">{{value._source.cast_name}}</h3>
+        <p class="card-text">by 
+          <span class="card-author">{{value._source.og_name}}</span>
+        </p>
+      </div>
+    </div>
+  </div>
   </div>
 </template>
 
+
 <script>
-import HelloWorld from './components/HelloWorld.vue'
 
 export default {
-  name: 'App',
```

Comment:
I'm not sure if removing this will cause problems or not, but it's not common to see Vue files without a name. Maybe that's just convention though.